Customer.cs

```diff
@@ -21,5 +21,12 @@ public class Customer
         private DateTime _loginDate;
         private DateTime _exitDate;
         private string _reservationType;
+
+        private string reservationSelection(int reservationNo)
```

Comment:
Başta ikisini(SelectedIndex, SelectedItem) birlikte almak mantıklı gelmişti ama sonra reservationSelection()'ına tek bir iş yaptırmak için combobox'ın SelectedIndex' ini alırken SelectedItem'ini almadım.
Database kısmına koyacağım SelectedItem'ı ayrıca sınıf üzerinden _reservationType değerine atamayı düşünüyorum. 
Bu kısım, form da newUserBooking bir kısmındaydı. Method olarak o kısımın içinde çağıracağım.